lib/exposure/exposure.dart

```diff
@@ -0,0 +1,5 @@
+abstract class Exposure<T> {
+  final List<T> items;
+  const Exposure(this.items);
+  Future<List<T>> getExposures(String data);
```

Comment:
Might just be nice to add some comments about what subclasses should be implementing and what the structure of the `data` looks like